addons/web_editor/static/src/js/editor/add_snippet_dialog.js

```diff
@@ -194,6 +194,8 @@ export class AddSnippetDialog extends Component {
             // Replace the snippet with an image preview if one exists.
             const imagePreview = snippet.imagePreview || originalSnippet?.imagePreview;
             if (imagePreview) {
+                // Enforce no-padding for image previews
+                clonedSnippetEl.style.setProperty('padding', '0', 'important');
```

Comment:
@stefanorigano We have to use double quotes... sorry for that

Reply:
Since you added this line, I think you can remove
```scss
            [data-snippet="s_map"], [data-snippet="s_google_map"] {
                padding: 0 !important;
            }
``` 
in the css

Reply:
done 👍